src/pages/Script/index.tsx

```diff
@@ -0,0 +1,288 @@
+      title: i18n.t('scripts.capacity_of_deployed_cells'),
+      content: <DecimalCapacity value={localeNumberString(shannonToCkb(capacityOfDeployedCells))} />,
+    },
+    {
+      title: '',
+      content: '',
+    },
+    {
+      title: i18n.t('scripts.capacity_of_referring_cells'),
+      content: <DecimalCapacity value={localeNumberString(shannonToCkb(capacityOfReferringCells))} />,
+    },
+  ]
+
+  return items
+}
+
+const ScriptsTitleOverview = ({ scriptInfo }: { scriptInfo: ScriptInfo }) => {
+  const dispatch = useDispatch()
+  return (
+    <div className={styles.scriptsTitleOverviewPanel}>
+      <OverviewCard items={getScriptInfo(scriptInfo, dispatch)} hideShadow />
+    </div>
+  )
+}
+
+const initScriptInfo: ScriptInfo = {
+  scriptName: '',
+  scriptType: '',
+  typeId: 0,
+  codeHash: '',
+  hashType: '',
+  capacityOfDeployedCells: '',
+  capacityOfReferringCells: '',
+  countOfTransactions: 0,
+  countOfDeployedCells: 0,
+  countOfReferringCells: 0,
+  pageOfTransactions: 1,
+  pageOfDeployedCells: 1,
+  pageOfReferringCells: 1,
+  sizeOfTransactions: 10,
+  sizeOfDeployedCells: 10,
+  sizeOfReferringCells: 10,
+}
+
+const seekScriptName = (codeHash: string, hashType: string): string =>
+  scriptHashNameMap.has(`${codeHash}_${hashType}`)
+    ? scriptHashNameMap.get(`${codeHash}_${hashType}`)!
+    : i18n.t('scripts.unnamed_script')
+
+export const ScriptPage = () => {
+  const history = useHistory()
+  const { search } = useLocation()
+  const { codeHash, hashType, tab } = useParams<{ codeHash: string; hashType: string; tab: ScriptTabType }>()
+  const anchor = (tab ?? 'transactions') as ScriptTabType
+  const [activeTab, setActiveTab] = useState<ScriptTabType>(anchor)
+  const parsed = queryString.parse(search)
+  const currentPage = parsePageNumber(parsed.page, PageParams.PageNo)
+  const pageSize = parsePageNumber(parsed.size, PageParams.PageSize)
+  initScriptInfo.codeHash = codeHash
+  initScriptInfo.hashType = hashType
+  initScriptInfo.scriptName = seekScriptName(codeHash, hashType)
+  const [scriptInfo, setScriptInfo] = useState<ScriptInfo>(initScriptInfo)
+
+  const { status, data: resp } = useQuery<AxiosResponse>(['scripts_general_info', codeHash, hashType], () =>
+    v2AxiosIns.get(`scripts/general_info`, {
+      params: {
+        code_hash: codeHash,
+        hash_type: hashType,
+      },
+    }),
+  )
+
+  useEffect(() => {
+    if (status === 'success' && resp) {
+      const response = toCamelcase<Response.Response<ScriptGeneralInfoResponse>>(resp?.data)
+      const { data } = response!
+
+      setScriptInfo(si => ({
+        ...si,
+        scriptType: data.scriptType,
+        typeId: data.typeId,
+        capacityOfDeployedCells: data.capacityOfDeployedCells,
+        capacityOfReferringCells: data.capacityOfReferringCells,
+        countOfTransactions: data.countOfTransactions,
+        countOfDeployedCells: data.countOfDeployedCells,
+        countOfReferringCells: data.countOfReferringCells,
```

Comment:
What if read these states from `resp` directly instead of updating `scriptInfo` and reading them from `scriptInfo`

Reply:
> What if read these states from `resp` directly instead of updating `scriptInfo` and reading them from `scriptInfo`

refactor to read from `resp` directly